### DISCUSSION TITLE
How to check if the branch is protected

### DISCUSSION THREAD

**Discussion by maczech** [hub4j/github-api#1036] [Q&A] [answered]
Hi,
I've noticed that both GHBranch.isProtected and GHBranch.getProtection are deprecated. What is the best choice to check if the branch is protected or not?

**Comment by bitwiseman** [comment, marked as answer]
Beta APIs are marked as deprecated to make it clear they may change without warning.  Otherwise, they can be used. See #1003 if you have any suggestion on how to make this clearer.

**Reply by maczech** [reply]
Thank you for making it clear
